contracts/test/unit/credit/CreditManagerV3.unit.t.sol

```diff
@@ -242,7 +242,8 @@ contract CreditManagerV3UnitTest is TestHelper, ICreditManagerV3Events, BalanceH
             t.mint(creditAccount, balance * ((i + 2) % 5));
 
             /// sets price between $0.01 and $60K
-            uint256 randomPrice = (uint256(keccak256(abi.encode(numberOfTokens, i, balance))) % 600_0000) * 10 ** 6;
+            uint256 randomPrice =
+                (uint256(keccak256(abi.encode(numberOfTokens, i, balance))) % (6000000 - 1) + 1) * 10 ** 6;
```

Comment:
the old line occasionally returned a price of 0 which caused price oracle mock to revert